base/applications/cmdutils/more/more.c

```diff
@@ -49,7 +49,7 @@ HANDLE hStdIn, hStdOut;
 HANDLE hKeyboard;
 
 /* Enable/Disable extensions */
-BOOL s_bEnableExtensions = TRUE; // FIXME: By default, it should be FALSE.
+BOOL s_bEnableExtensions = FALSE;
```

Comment:
See comment below regarding having EnableExtensions in the registry, yet.

Reply:
Reverted.